src/content/docs/style-guide/quick-reference/capitalization.mdx

```diff
@@ -22,6 +22,10 @@ Use sentence case for headings. This includes category headings and document tit
 * If the heading is a code term, such as a variable or function, then capitalize it exactly as it's used in the code; for example: [`noticeError`](/docs/browser/new-relic-browser/browser-agent-spa-api/notice-error).
 * If the heading includes a colon, follow the [Microsoft Style Guide](https://docs.microsoft.com/en-us/style-guide/punctuation/colons#in-titles-and-headings) for titles and headings, and capitalize the first word that appears after the colon; for example: [APM Error profiles: Troubleshoot trends](/docs/apm/applications-menu/error-analytics/apm-error-profiles-troubleshoot-trends).
 
+<Callout variant="important">
+  Use sentence case for graphical illustrations such as diagrams and figures. Explore our [Screenshots and images](https://docs.newrelic.com/docs/style-guide/writing-guidelines/screenshots-images/#graphics) document for more information on our image guidelines.
```

Comment:
For internal links, strip out the `https://docs.newrelic.com`. 

That URL should be: `/docs/style-guide/writing-guidelines/screenshots-images/#graphics`

For external sites, like the Microsoft Style Guide, you'll need the full URL, of course.